xpdacq/beamtime.py

```diff
@@ -329,7 +329,7 @@ def _scanplan_name_parser(self, sp_name):
         _Tramp_optional_params = ['det', 'subs_dict']
         _tseries_optional_params = ['det', 'subs_dict']
          
-        parsed_object = sp_name.split('_', maxsplit = 4)
+        parsed_object = sp_name.split('_', maxsplit = 4+3) # 3 is error tolerance
```

Comment:
why do you specify ``maxsplit``?  Doesn't it add complication if we want to add different scan types later?  If it is not used for some reason, they I suggest just removing it.